targets/TARGET_NUVOTON/TARGET_M480/crypto/crypto-misc.h

```diff
@@ -32,24 +32,27 @@ void crypto_uninit(void);
 void crypto_zeroize(void *v, size_t n);
 void crypto_zeroize32(uint32_t *v, size_t n);
 
-/* Acquire/release ownership of AES H/W */
-/* NOTE: If "acquire" succeeds, "release" must be done to pair it. */
-bool crypto_aes_acquire(void);
+/* Acquire/release ownership of crypto sub-module
+ * 
+ * \param blocking  false for non-blocking
```

Comment:
You don't need the blocking parameter, and the entire choice of locking primitive is on the basis the AES/DES/ECC always block, and SHA never does, so it would be clearer to take it out.

And the distinction could then just be in the function names:

    void crypto_aes_acquire(void);
    bool crypto_sha_try_acquire(void);

Reply:
@kjbracey-arm Fix as above